src/WCSimDetectorMessenger.cc

```diff
@@ -805,6 +810,17 @@ void WCSimDetectorMessenger::SetNewValue(G4UIcommand* command,G4String newValue)
 		G4cout << "Setting Gadolinium doping concentration: " << newValue << "percent" << G4endl;
             WCSimDetector->AddDopedWater(DopingConcentration->GetNewDoubleValue(newValue));
 	}
+  
+  if(command == BGOPlacement) {
+    if (newValue == "true") {
```

Comment:
```suggestion
    if (BGOPlacement->GetNewBoolValue(newValue)) {
```
This should work in all cases of what someone puts in the mac file (e.g. `TRUE`, `true`, `True`, `1`, ...) (please do test!) but I'm not sure that your version will